src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const Action of actions) {
+    switch (Action.type) {
```

Comment:
```suggestion
  for (const action of actions) {
    switch (action.type) {
```
general convention to name variables in camelCase in js. so first word should be from lower case

Reply:
Thanks, I'll be attention